ansible/playbooks/roles/containerd/handlers/main.yml

```diff
@@ -0,0 +1,9 @@
+- name: Restart Containerd
```

Comment:
Containerd is written starting with lowercase, please see https://containerd.io/ or https://github.com/containerd/containerd

So I would replace `Containerd` -> `containerd` in all files 😉